assets/css/detours/_detour_modal.scss

```diff
@@ -1,11 +1,11 @@
 .c-detour-modal {
   inset: 0;
   position: fixed;
-  transition: all 0.3s;
   z-index: map-get($z-page-layout-context, "detour-modal");
 }
 
-.c-detour-modal__hidden {
+.c-detour-modal__transition-exit-active {
+  transition: all 0.3s;
   transform: scale(1.02) translate(0px, -50px);
   opacity: 0;
   z-index: map-get($z-page-layout-context, "detour-modal-hidden");
```

Comment:
Now that the modal gets removed from the DOM instead of just hidden with 0 opacity, I think we can get rid of this z-index as well (and probably the `detour-modal-hidden` thing from `$z-page-layout-context`)

Reply:
ea834db3